homework/index.js

```diff
@@ -30,14 +36,102 @@
     return elem;
   }
 
+  function createOptions(repoNames, select) {
+    createAndAppend('option', select, { text: 'Select module', disabled: 'disabled' });
+    repoNames.sort((a, b) => a.name.localeCompare(b.name));
+    repoNames.forEach((repository, i) => {
+      createAndAppend('option', select, {
+        text: repository.name,
+        value: repository.name,
+      });
+    });
+  }
+
+  function renderRepoInformation(repo, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repo}`;
+    fetchJSON(url, (err, repoInfos) => {
+      if (err) {
+        renderError(err);
+        return;
+      }
+
+      ul.innerHTML = ' ';
+      const li = createAndAppend('li', ul);
+      const table = createAndAppend('table', li, { class: 'infoTable' });
```

Comment:
CSS names should be in kebab-case by convention: `info-table`. This holds true for several places in your code.